data/views/redfish-1.0/ucs.1.0.0.processorcollection.json

```diff
@@ -0,0 +1,16 @@
+{
+    "@odata.context" : "<%= basepath %>/$metadata#Systems/Links/Members/<%= identifier %>/Processors/#entity",
+    "@odata.id": "<%= url %>",
+    "@odata.type": "#Procssors.1.0.0.ProcessorsCollection",
```

Comment:
"Procssors" typo. 
One question: why are these fields different from both https://github.com/RackHD/on-http/blob/master/data/views/redfish-1.0/wsman.1.0.0.processorcollection.json and https://github.com/RackHD/on-http/blob/master/data/views/redfish-1.0/redfish.1.0.0.processorcollection.json ?